### PR TITLE
Fix or create new failing Django test

### PR DESCRIPTION
**Pull request by mumerfarooq000**
fixed the failing test and create new for the one that needed and was missing

a detailed description will be edited once finalized the PR

### CODE REVIEW

File: ecommerce/extensions/catalogue/tests/test_convert_course.py [hunk 1]
```diff
@@ -1,86 +1,90 @@
...
+#         self.assertEqual(len(new_seats), 0)
+#
+#     @ddt.data('honor_to_audit', 'audit_to_honor')
+#     def test_without_seats(self, direction):
+#         """Verify that the command fails when the course does not have the correct seat type."""
+#         course = CourseFactory()
+#
+#         call_command('convert_course', course.id, commit=True, direction=direction, partner=self.partner)
+#
+#         self.assertEqual(len(course.seat_products), 0)
```
Comment (danialmalik) [inline ecommerce/extensions/catalogue/tests/test_convert_course.py:90]: Remove the commented code

**Comment by danialmalik** [comment]
@mumerfarooq000 since we are not using the translation tests, remove that job from the file: `.github/workflows/pull_request_ci.yml`

image

File: ecommerce/extensions/catalogue/tests/test_convert_course.py (hunk 1 above)
Comment (mumerfarooq000) [reply ecommerce/extensions/catalogue/tests/test_convert_course.py:90]: have to remove the file then

**Comment by mumerfarooq000** [comment]
@danialmalik done with changes you asked

**Comment by danialmalik** [comment]
> @danialmalik done with changes you asked

@mumerfarooq000 thanks

**Pull request merged**